packages/db/src/translate/providers.ts

```diff
@@ -11,6 +12,29 @@ export const encryptValueWithKMS = (
   return encryptFunction(providerKmsKeyId, value);
 };
 
+export const translatePostgresProviderToApiProvider = (
+  record: PostgresProviderRecord
+): ApiProvider => {
+  const apiProvider = {
+    id: record.name,
+    cmKeyId: record.cm_key_id,
+    certificateUri: record.certificate_uri,
+    privateKey: record?.private_key,
+    globalConnectionLimit: record.global_connection_limit,
+    port: record.port,
+    host: record.host,
+    protocol: record.protocol,
+    createdAt: record.created_at.getTime(),
+    updatedAt: record.updated_at.getTime(),
+    username: record.username,
+    password: record.username,
+  } as ApiProvider;
+  if (record.username || record.password) {
+    apiProvider.encrypted = true;
```

Comment:
This is needed because behavior in-code expecting the API to have this set to true is present in-code.  Perhaps more importantly, making it implicit would be a breaking change.    

I've added a ticket in the interest of scope-limiting this PR/for discussion in this week's arch meeting.

Reply:
Maybe consider adding a comment about this as well in the code in case others don't find the comment from the PR ?

Reply:
Hmm, that's a good idea.  Will do.